doc/examples/fast_streamline_search.py

```diff
@@ -0,0 +1,203 @@
+"""
+======================
+Fast Streamline Search
+======================
+
+This example explains how Fast Streamline Search [StOnge2021]_
+can be used to find all similar streamlines.
+
+First import the necessary modules.
+"""
+
+import numpy as np
+
+from dipy.data import get_target_tractogram_hcp, get_two_hcp842_bundles
+from dipy.io.streamline import load_trk
+from dipy.segment.fss import FastStreamlineSearch, nearest_from_matrix_row
+from dipy.viz import actor, window
+
+"""
+Download and read data for this tutorial
+"""
+
```

Comment:
Please add a call to the fetches before in case the data was not downloaded by the user
```
from dipy.data import fetch_bundle_atlas_hcp842, fetch_target_tractogram_hcp                         

fetch_bundle_atlas_hcp842()
fetch_target_tractogram_hcp()
```

Reply:
done